src/sampler.jl

```diff
@@ -0,0 +1,67 @@
+#=
+    `MVNSampler` is used to draw from multivariate normal distribution
+=#
+
+import Base: ==
+
+immutable MVNSampler{TM<:Real,TS<:Real,TQ<:LinAlg.BlasReal}
+    mu::Vector{TM}
+    Sigma::Matrix{TS}
+    Q::Matrix{TQ}
+end
+
+function MVNSampler{TM<:Real,TS<:Real}(mu::Vector{TM}, Sigma::Matrix{TS})
+    TOL1 = 1e-10  # some small value
+    TOL2 = 1e-10  # some small value

```

Comment:
Sorry, I have no idea what "optimal" means. 

Does it mean the tolerance should be relative value?